qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -38,7 +40,25 @@
 
 
 class IBMQFactory:
-    """Factory and account manager for IBM Quantum Experience."""
+    """Factory and account manager for IBM Quantum Experience.
+
+    When saving an account, via :meth:`save_account`, you should specify a default
```

Comment:
I don't know if one _should_ specify a default. Users that only have access to the open h/g/p certainly don't need to do this.